src/Util.cc

```diff
@@ -989,6 +989,38 @@ const common::Mesh *optimizeMesh(const sdf::Mesh &_meshSdf,
   return optimizedMesh;
 }
 
+math::AxisAlignedBox meshAxisAlignedBox(sdf::Mesh _sdfMesh)
```

Comment:
consider also returning `std::optional` to indicate that the axis aligned box can not be computed

Reply:
Addressed in https://github.com/gazebosim/gz-sim/pull/2821/commits/84e70e798d227bac99f250b417344fec739833f2